### PR TITLE
[Tests] Add stylelint for public/css/*.css

### LINKED ISSUE #299: public/css/*.css needs to be linted
We are adding linting for all main files relating to the website. We have already implemented eslint for the backend scripts and linting for the HTML templates and front-end JS are in the works.

We should also be linting the main css files that are used for the site to ensure they also keep a good code style standard going forward.

### PR DESCRIPTION
**Pull request by MattIPv4**
Resolves #299 by bringing in stylelint with a custom ruleset.
Brings both existing css files up-to-standard.
Adds stylelint to test script.

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -1,218 +1,238 @@
...
+    width: 200px;
+    z-index: 0;
+  }
+
+  #forkongithub a {
+    box-shadow: 4px 4px 10px rgba(0, 0, 0, .8);
+    left: -45px;
+    position: absolute;
+    top: 30px;
+    -webkit-transform: rotate(315deg);
```
Comment (drewfreyling) [inline public/css/main.css]: This code reminds me that we should probably try something like autofixer in the future.

Comment (MattIPv4) [reply public/css/main.css]: Agreed. I think with the website redesign I'm probably going to start from scratch on the styling using SCSS, so I can ensure that everything gets autoprefixed as part of the build process :)

File: public/css/theme.css [hunk 2]
```diff
@@ -70,15 +70,15 @@ body {
 p#name a{
   bottom: 8px;
   color: #007ba8;
-  font-family: arial, helvetica;
+  font-family: arial, helvetica, sans-serif;
```
Comment (PeterDaveHello) [inline public/css/theme.css]: Is this part of the stylelint?

Comment (MattIPv4) [reply public/css/theme.css]: Yes, having a generic fallback for the font family.

**Pull request merged**